modules/policy-as-code-create-portal.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-security-policies/custom-security-policies.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="policy-as-code-create-portal_{context}"]
+= Creating policies in code by using the {product-title-short} portal
+
+[role="_abstract"]
+You can create new policies in code by using the {product-title-short} portal to save existing policies as YAML files.
+
+To create a new policy in code by using the {product-title-short} portal:
+
+. In the *Policy Management* page, create a new policy or clone a default policy.
++
+[NOTE]
+====
+You must clone a default policy before you can save it as a CR.
+====
+. In the row listing the policy, click the overflow menu, {kebab}, and then select *Save as Custom Resource*. To save multiple policies at one time, you can select them and click *Bulk actions* -> *Save as Custom Resources*.
++
+[IMPORTANT]
+====
+If you are committing the saved custom resource to a source control repository, replace the policy name in the `policyName` field to avoid overwriting existing policies.
```

Comment:
I remember discussing this, but I don't remember coming to this conclusion, because I would personally recommend against renaming the policy.  The goal (imo) is for the config controller to "take over" control of the policy once it's saved as a CR.